eng/pipelines/runtime.yml

```diff
@@ -1772,7 +1772,7 @@ extends:
           jobParameters:
             testGroup: innerloop
             nameSuffix: Mono_Interpreter_LibrariesTests
-            buildArgs: -s mono+libs+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true
+            buildArgs: -s mono+libs+libs.tests -rc Release -c $(_BuildConfig) /p:ArchiveTests=true
```

Comment:
I see, you're still giving the option of choosing the -c for the runtime, even though -rc assumes Debug on libraries. They don't collide with each other, right?

Reply:
Yeah this works fine. I generally prefer the explicitness of passing `-c` always in case we want to change it. It's not necessary here, just preference of explicitness (like the rest of the pipelines).